### PR TITLE
localization of numbers in date picker calender widget

### PR DESCRIPTION
**Pull request by devgurjar**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/clientlibs/site/js/datepickerwidget.js [hunk 1]
```diff
@@ -1086,6 +1086,10 @@ if (typeof window.DatePickerWidget === 'undefined') {
         if (clearText) {
             defaultOptions.locale.clearText = clearText;
         }
+        var zero = FormView.LanguageUtils.getTranslatedString(locale, "0");
```
Comment (rismehta) [inline ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/clientlibs/site/js/datepickerwidget.js:1089]: Please add a test case for this

**Review by rismehta — CHANGES_REQUESTED** [review]
check comments

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/clientlibs/site/js/datepickerwidget.js (hunk 1 above)
Comment (iamsudhanshu) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/clientlibs/site/js/datepickerwidget.js:1104]: @devgurjar To confirm - this was documented - but not implemented? https://github.com/adobe/aem-core-forms-components/pull/990/files#diff-762b951cd2955c5f45991f718c78ceeae34eb41c962569de26f1c6f753361ccfR44

Comment (iamsudhanshu) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/clientlibs/site/js/datepickerwidget.js:1104]: Can we add a test case for this part. Also - this is independent of current Jira - or customer requires this as part of the change as well?

Comment (devgurjar) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/clientlibs/site/js/datepickerwidget.js:1104]: This is independent of current jira, customer has not asked for this part

**Review by sakshi-arora1 — APPROVED** [review]

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/clientlibs/site/js/datepickerwidget.js (hunk 1 above)
Comment (devgurjar) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/clientlibs/site/js/datepickerwidget.js:1104]: I have added test cases for Zero as well

**Pull request merged**